src/System.Runtime.Extensions/System.Runtime.Extensions.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2013
-VisualStudioVersion = 12.0.21005.1
+# Visual Studio 14
+VisualStudioVersion = 14.0.25420.1
```

Comment:
I don't think it matters much, but I would undo these changes as they are irrelevant to your change.